.github/workflows/test.yml

```diff
@@ -16,6 +16,8 @@ jobs:
         include:
           - { python-version: "3.10", os: windows-latest }
           - { python-version: "3.10", os: macos-latest }
+          - { python-version: "pypy-3.8", os: windows-latest }
+          - { python-version: "pypy-3.8", os: macos-latest }
```

Comment:
PyPy tests are a bit slower than CPython, let's give them a bit of a headstart:

```suggestion
          - { python-version: "pypy-3.8", os: windows-latest }
          - { python-version: "pypy-3.8", os: macos-latest }
          - { python-version: "3.10", os: windows-latest }
          - { python-version: "3.10", os: macos-latest }
```

Reply:
I'm fairly certain that the order in which you declare them is ignored. I've tried reording jobs before and it didn't make any difference. That was a while ago though so it might have changed.

Reply:
Here's a comparison:

<table>
<tr>
	<td><a href=https://github.com/ultrajson/ultrajson/actions/runs/2518134150>yesterday</a>
	<td><a href=https://github.com/ultrajson/ultrajson/actions/runs/2521164008>now</a>
<tr>
	<td><img width="274" alt="image" src="https://user-images.githubusercontent.com/1324225/174450196-bdce460f-0848-41cb-9e82-6cfd3a8a3890.png">
	<td><img width="281" alt="image" src="https://user-images.githubusercontent.com/1324225/174450205-22364756-d52d-4914-beb0-98c3f0feb47d.png">
</table>